### DISCUSSION TITLE
无法正常启动

### DISCUSSION THREAD

**Discussion by llsa0508** [libnyanpasu/clash-nyanpasu#635] [Q&A]
第一次安装配置好后一切正常，关机重启电脑后无法正常启动应用了。求助
<img width="266" alt="Snipaste_2024-03-19_10-29-35" src="https://github.com/LibNyanpasu/clash-nyanpasu/assets/163955789/6111c2bc-f28f-41d6-b08f-e6eac9884822">

**Comment by mofater** [comment]
![截图 2024-04-14 18-36-22](https://github.com/LibNyanpasu/clash-nyanpasu/assets/27564158/e4be1d86-b1a0-4bdd-89ff-2921d4a86427)
我也是，windows坏了，心血来潮装了个Debian玩玩，不懂linux正在摸索，只会简单的安装程序，用dpkg安装后都调好了开机静默启动，系统代理，代理也检查可以用，但是重启后就报错了

**Reply by greenhat616** [reply]
这个应该是重启的时候没有正确释放单例检测锁。

cc @4o3F  可能需要改一下 Linux 下的单例检测

**Reply by 4o3F** [reply]
Linux下的单例检测是创建的abstract socket，按理说关闭后会被系统回收，暂时没找到原因，需要进一步debug下

**Reply by mofater** [reply]
似乎是静默启动导致的，关掉就可以正常运行了
      _,met$$$$$gg.          mofater@mo 
    ,g$$$$$$$$$$$$$$$P.       ---------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 12 (bookworm) x86_64 
 ,$$P'              `$$$.     Host: 82JQ Lenovo Legion R9000P2021H 
',$$P       ,ggs.     `$$b:   Kernel: 6.1.0-20-amd64 
`d$$'     ,$P"'   .    $$$    Uptime: 1 min 
 $$P      d$'     ,    $$P    Packages: 2014 (dpkg) 
 $$:      $$.   -    ,d$$'    Shell: bash 5.2.15 
 $$;      Y$b._   _,d$P'      Resolution: 2560x1600 
 Y$$.    `.`"Y$$$$P"'         DE: GNOME 43.9 
 `$$b      "-.__              WM: Mutter 
  `Y$$                        WM Theme: Adwaita 
   `Y$$.                      Theme: Adwaita [GTK2/3] 
     `$$b.                    Icons: Adwaita [GTK2/3] 
       `Y$$b.                 Terminal: gnome-terminal 
          `"Y$b._             CPU: AMD Ryzen 7 5800H with Radeon Graphics (16)  
              `"""            GPU: NVIDIA GeForce RTX 3060 Mobile / Max-Q 
                              Memory: 3030MiB / 15842MiB
